src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -41,7 +41,9 @@ export const thorchainApi: Swapper2Api = {
       }))
     }
 
-    return await getThorTradeQuote(input, rates).then(async firstQuote => {
+    return await getThorTradeQuote({
+      ...input,
```

Comment:
Remove spread?

Reply:
0dacf628d488b506f646236353b7b7db166a4191

Reply:
Nice, not sure why that was there!